tox.ini

```diff
@@ -4,14 +4,17 @@ skipsdist = True
 envlist = py36,py37,py38,py39
 
 [testenv]
-deps = .[dev]
+setenv =
+    PYTHONPATH = {toxinidir}
 commands =
         python -m pip install --upgrade pip
+        python -m pip install .[dev]
         pylint -j 0 -s n setup.py pytest_rts
         mypy setup.py pytest_rts --ignore-missing-imports
         safety check
         pytest \
           --cov=pytest_rts \
+          --cov-fail-under=93.25 \
```

Comment:
hey! [96% coverage is reached](https://github.com/F-Secure/pytest-rts/pull/93/checks?check_run_id=2139100355#step:4:290), why so low threshold? 😄